frontend/packages/core/src/AppLayout/search.tsx

```diff
@@ -153,26 +228,44 @@ const SearchField: React.FC = () => {
     setInputValue("");
   };
 
+  const handleOpen = () => {
+    setOpen(!open);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
   return (
-    <Container>
-      <Autocomplete
-        autoComplete
-        freeSolo
-        selectOnFocus
-        size="small"
-        inputValue={inputValue}
-        renderInput={Input}
-        renderOption={renderResult}
-        onInputChange={onInputChange}
-        open={showOptions}
-        onOpen={onOptionsOpen}
-        onClose={onOptionsClose}
-        options={options}
-        filterOptions={filterResults}
-        getOptionLabel={x => x.label}
-        ListboxProps={{ style: { backgroundColor: fade(theme.palette.secondary.main, 0.75) } }}
-      />
-    </Container>
+    <Grid container alignItems="center">
+      {open ? (
+        <ClickAwayListener onClickAway={handleClose}>
+          <Autocomplete
+            autoComplete
+            selectOnFocus
+            size="small"
+            inputValue={inputValue}
+            renderInput={Input}
+            renderOption={renderResult}
+            onInputChange={onInputChange}
+            open={showOptions}
+            onOpen={onOptionsOpen}
+            onClose={onOptionsClose}
+            options={options}
+            filterOptions={filterResults}
+            getOptionLabel={x => x.label}
+            PopperComponent={renderPopper}
+            popupIcon={<CustomCloseIcon/>}
+            forcePopupIcon={showOptions ? true: false}
```

Comment:
without it, the close icon will always appear before a user has selected the input field. based on the designs, we want the icon to appear once the user has clicked into the input field/started typing.